modules/lnd.nix

```diff
@@ -276,9 +302,14 @@ in {
     # Advantages of manually pre-generating certs:
     # - Reduces dynamic state
     # - Enables deployment of a mesh of server plus client nodes with predefined certs
-    nix-bitcoin.generateSecretsCmds.lnd = ''
+    nix-bitcoin.generateSecretsCmds.lnd = let
+      extraAltNames = builtins.concatStringsSep "," (
+        (map (ip: "IP:${ip}") cfg.certificate.extraIPs) ++
+        (map (domain: "DNS:${domain}") cfg.certificate.extraDomains)
+      );
+    in ''
       makePasswordSecret lnd-wallet-password
-      makeCert lnd '${optionalString (cfg.rpcAddress != "localhost") "IP:${cfg.rpcAddress}"}'
+      makeCert lnd '${extraAltNames}"}'
```

Comment:
I think `localhost` should always be added, as lncli/loop/pool is broken if not.